meson.build

```diff
@@ -680,6 +680,11 @@ if id_result.returncode() == 0
                         '        Your build will result in an group table setup that is incompatible with the local system.')
         endif
 endif
+if nobody_user != nobody_group
+        message('WARNING:\n' +
+                '        The configured user name "@0@" and group name "@0@" of the nobody user/group are not equivalent.\n'.format(nobody_user, nobody_group) +
+                '        Please re-check that both "nobody-user" and "nobody-group" options are correctly set.')
```

Comment:
Actually on debian the names are "nobody" and "nogroup", and we explicitly support that.

Reply:
Oh, thank you. I did not know that. Indeed, nogroup is mentioned in UIDS-GIDS.md.